core/src/main/java/org/ehcache/core/statistics/CacheStatistics.java

```diff
@@ -20,43 +20,119 @@
 
 /**
  * All statistics relative to a cache and its underlying tiers.
- *
- * @author Henri Tremblay
  */
 public interface CacheStatistics {
 
+  /**
+   * List of statistics tracked on this cache
+   *
+   * @return a map of statistics per name
+   */
   Map<String, TypedValueStatistic> getKnownStatistics();
 
   /**
+   * Map of tier statistics on this cache. Per tier name
+   *
+   * @return tier statistics per tier name
+   */
+  Map<String, TierStatistics> getTierStatistics();
```

Comment:
I am wondering if we are better to have an enum as a key instead of some random string here ? since this interface is in ehache core... perhaps also an enum would help for comparison if needed.